icecreamshop/src/main/java/com/summerschool/icecreamshop/model/dto/ProductDTO.java

```diff
@@ -0,0 +1,129 @@
+package com.summerschool.icecreamshop.model.dto;
```

Comment:
pachetul dto este duplicat, apare odata in com.summerschool.icecreamshop si inca o data in com.summerschool.icecreamshop.model